### PR TITLE
new readme file for new repository

### PR DESCRIPTION
**Pull request by iwanlevin**

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
-            var workingGrade = workingRow.getElementsByTagName('input')[1].value;
-            workingGrade = Math.round(workingGrade / outOf * 100);
-    
-            students.push(new Student(workingId, workingGrade));
-    }
+  var studentCount = body.getElementById('StudentCount').value;
+  var outOf = body.getElementById('OutOf').value;
+
+  for (var i = 0; i < studentCount; i++) {
+    //var workingRow = body.getElementsByTagName('tr').item(i); <- Don't need this anymore
```
Comment (Ianden) [inline content.js]: We get `workingId` relative to `workingRow`. Technically we could combine the two lines into one, though this approach is more readable.

Comment (Ianden) [inline content.js]: Oh, now I see what you did. You're right, that works better.

Comment (Ianden) [inline content.js]: We are still getting `workingGrade` relative to `workingRow` on line 22. We could shorten it your way, though I think using `workingRow` looks more readable.

**Comment by Ianden** [comment]
Modified the README a bit to be more relevant to the Chrome extension. Let me know what you think of the comment I made in content.js.

File: content.js (hunk 1 above)
Comment (iwanlevin) [inline content.js]: That's true it is a lot more readable I guess in the end it'll be, do we want efficiency for large lists of students or do we want readability with more code either one will work

**Comment by iwanlevin** [comment]
New README looks good and left a comment in content.js

File: content.js (hunk 1 above)
Comment (Ianden) [inline content.js]: An extra variable declaration will make no difference to efficiency. Merging with a new `cleanup` branch so we can accumulate minor fixes, and test on Monday before merging with `master`.

**Pull request merged**